designer/client/src/components/graph/Graph.tsx

```diff
@@ -281,9 +281,12 @@ export class Graph extends React.Component<Props> {
         this.panAndZoom.fitContent(area, this.viewport);
     }, 250);
 
-    fitToNodes = (nodeIds: NodeId[] = []): void => {
-        const cells = this.graph.getCells().filter((c) => nodeIds.includes(c.id as string));
-        this.fit(cells);
+    fitToNode = (nodeId: NodeId): void => {
+        const cellsToFit = this.graph.getCells().filter((c) => c.id === nodeId);
+        const area = cellsToFit?.length ? this.graph.getCellsBBox(cellsToFit) : this.processGraphPaper.getContentArea();
```

Comment:
1. I think the `cellsToFit?` null check is redundant since `.filter` will return empty array if it doesnt find anything
2. Isn't it guaranteed that we find exactly one node? If so, we could use `.find` instead of `filter`

Reply:
1. Changed ✔️ 
2. Good idea, maybe it shows our intentions in a clearer way, the only downside is `this.graph.getCellsBBox([cellToFit])` but IMO it's acceptable